prac_06/programming_language.py

```diff
@@ -0,0 +1,18 @@
+"""CP1404/CP5632 Practical - Programming language class"""
+
+
+class ProgrammingLanguage(object):
+    """Represent a programming language Object"""
+    def __init__(self, name='', typing="", reflection="", year=0):
+        """Initialize a programming language instance"""
+        self.name = name
+        self.typing = typing
+        self.reflection = reflection
+        self.year = year
+
+    def is_dynamic(self):
+        """Determine if instance is dynamic"""
+        return True if self.typing.lower() == "dynamic" else False
```

Comment:
See https://github.com/CP1404/Starter/wiki/Programming-Patterns#working-with-booleans

Reply:
Ah. I noticed other students making this error too. Thanks for pointing this out.

Reply:
It's not an "error", but rather, there are several ways of doing it, and I think the easiest and most efficient is to return the result of the expression.